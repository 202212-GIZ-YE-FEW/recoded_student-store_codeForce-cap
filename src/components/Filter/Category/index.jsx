import { withTranslation } from "next-i18next"
import { useState } from "react"

import ProductList from "@/components/ProductList"

import { useGeneralListings } from "@/utils/store"

// import products from "@/components/ProductList/products"
import Buttons from "../Buttons"

function CategoryFilter() {
  const { data } = useGeneralListings()
  const filters = [
    {
      filter: "Book",
      image: "/images/Book.png",
      alt: "Book",
    },
    {
      filter: "Furniture",
      image: "/images/Furniture.png",
      alt: "Furniture",
    },
    {
      filter: "Electronics",
      image: "/images/Electronic.png",
      alt: "Electronics",
    },
    {
      filter: "Two-wheeler",
      image: "/images/Two-wheeler.png",
      alt: "Two-wheeler",
    },
  ]

  const [selectedFilter, setSelectedFilter] = useState("")

  const handleCategoryFilter = (filter) => setSelectedFilter(filter)

  return (
    <div>
      <Buttons filters={filters} handleCategoryFilter={handleCategoryFilter} />
      <ProductList selectedFilter={selectedFilter} products={data} />
    </div>
  )
}

export default withTranslation("index")(CategoryFilter)
