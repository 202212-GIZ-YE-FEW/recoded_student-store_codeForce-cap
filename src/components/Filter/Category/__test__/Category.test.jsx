import renderer from "react-test-renderer"

import Category from ".."

it("renders correctly", () => {
  const tree = renderer.create(<Category />).toJSON()
  expect(tree).toMatchSnapshot()
})
