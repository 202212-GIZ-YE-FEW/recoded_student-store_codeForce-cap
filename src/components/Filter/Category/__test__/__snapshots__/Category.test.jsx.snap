// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="max-w-[1250px] mx-auto my-[50px]"
>
  <div>
    <div>
      <button
        onClick={[Function]}
      >
        All
      </button>
      <button
        className="filter Book"
        onClick={[Function]}
      >
        <img
          alt="Book"
          data-nimg="1"
          decoding="async"
          height={137}
          loading="lazy"
          onError={[Function]}
          onLoad={[Function]}
          src="/_next/image?url=%2Fimages%2FBook.png&w=384&q=75"
          srcSet="/_next/image?url=%2Fimages%2FBook.png&w=256&q=75 1x, /_next/image?url=%2Fimages%2FBook.png&w=384&q=75 2x"
          style={
            Object {
              "color": "transparent",
            }
          }
          width={137}
        />
      </button>
      <button
        className="filter Furniture"
        onClick={[Function]}
      >
        <img
          alt="Furniture"
          data-nimg="1"
          decoding="async"
          height={137}
          loading="lazy"
          onError={[Function]}
          onLoad={[Function]}
          src="/_next/image?url=%2Fimages%2FFurniture.png&w=384&q=75"
          srcSet="/_next/image?url=%2Fimages%2FFurniture.png&w=256&q=75 1x, /_next/image?url=%2Fimages%2FFurniture.png&w=384&q=75 2x"
          style={
            Object {
              "color": "transparent",
            }
          }
          width={137}
        />
      </button>
      <button
        className="filter Electronics"
        onClick={[Function]}
      >
        <img
          alt="Electronics"
          data-nimg="1"
          decoding="async"
          height={137}
          loading="lazy"
          onError={[Function]}
          onLoad={[Function]}
          src="/_next/image?url=%2Fimages%2FElectronic.png&w=384&q=75"
          srcSet="/_next/image?url=%2Fimages%2FElectronic.png&w=256&q=75 1x, /_next/image?url=%2Fimages%2FElectronic.png&w=384&q=75 2x"
          style={
            Object {
              "color": "transparent",
            }
          }
          width={137}
        />
      </button>
      <button
        className="filter Two-wheeler"
        onClick={[Function]}
      >
        <img
          alt="Two-wheeler"
          data-nimg="1"
          decoding="async"
          height={137}
          loading="lazy"
          onError={[Function]}
          onLoad={[Function]}
          src="/_next/image?url=%2Fimages%2FTwo-wheeler.png&w=384&q=75"
          srcSet="/_next/image?url=%2Fimages%2FTwo-wheeler.png&w=256&q=75 1x, /_next/image?url=%2Fimages%2FTwo-wheeler.png&w=384&q=75 2x"
          style={
            Object {
              "color": "transparent",
            }
          }
          width={137}
        />
      </button>
    </div>
    <div>
      <div
        className="carts grid xl:grid-cols-4 lg:grid-cols-4 md:grid-cols-2 sm:grid-cols-1 m-auto w-full"
      >
        <div
          className="cart mx-3 mb-10 border rounded-lg shadow-lg"
        >
          <img
            alt="Moog Synthesizer"
            className="rounded-t-lg shadow-lg w-full"
            data-nimg="1"
            decoding="async"
            height={211.41}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/_next/image?url=%2Fimg.jpg&w=640&q=75"
            srcSet="/_next/image?url=%2Fimg.jpg&w=384&q=75 1x, /_next/image?url=%2Fimg.jpg&w=640&q=75 2x"
            style={
              Object {
                "color": "transparent",
              }
            }
            width={258}
          />
          <div
            className="mx-3 text-center"
          >
            <div
              className="info flex justify-between my-4 mx-3"
            >
              <div
                className="text-left"
              >
                <h2
                  className="font-semibold"
                >
                  Moog Synthesizer
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Books
                </p>
              </div>
              <div>
                <h2
                  className="font-extrabold text-xl"
                >
                  $
                  468.00
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Istanbul / Turkey
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          className="cart mx-3 mb-10 border rounded-lg shadow-lg"
        >
          <img
            alt="Moog Synthesizer"
            className="rounded-t-lg shadow-lg w-full"
            data-nimg="1"
            decoding="async"
            height={211.41}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/_next/image?url=%2Fimg.jpg&w=640&q=75"
            srcSet="/_next/image?url=%2Fimg.jpg&w=384&q=75 1x, /_next/image?url=%2Fimg.jpg&w=640&q=75 2x"
            style={
              Object {
                "color": "transparent",
              }
            }
            width={258}
          />
          <div
            className="mx-3 text-center"
          >
            <div
              className="info flex justify-between my-4 mx-3"
            >
              <div
                className="text-left"
              >
                <h2
                  className="font-semibold"
                >
                  Moog Synthesizer
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Furniture
                </p>
              </div>
              <div>
                <h2
                  className="font-extrabold text-xl"
                >
                  $
                  468.00
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Istanbul / Turkey
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          className="cart mx-3 mb-10 border rounded-lg shadow-lg"
        >
          <img
            alt="Moog Synthesizer"
            className="rounded-t-lg shadow-lg w-full"
            data-nimg="1"
            decoding="async"
            height={211.41}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/_next/image?url=%2Fimg.jpg&w=640&q=75"
            srcSet="/_next/image?url=%2Fimg.jpg&w=384&q=75 1x, /_next/image?url=%2Fimg.jpg&w=640&q=75 2x"
            style={
              Object {
                "color": "transparent",
              }
            }
            width={258}
          />
          <div
            className="mx-3 text-center"
          >
            <div
              className="info flex justify-between my-4 mx-3"
            >
              <div
                className="text-left"
              >
                <h2
                  className="font-semibold"
                >
                  Moog Synthesizer
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Furniture
                </p>
              </div>
              <div>
                <h2
                  className="font-extrabold text-xl"
                >
                  $
                  468.00
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Istanbul / Turkey
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          className="cart mx-3 mb-10 border rounded-lg shadow-lg"
        >
          <img
            alt="Moog Synthesizer"
            className="rounded-t-lg shadow-lg w-full"
            data-nimg="1"
            decoding="async"
            height={211.41}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/_next/image?url=%2Fimg.jpg&w=640&q=75"
            srcSet="/_next/image?url=%2Fimg.jpg&w=384&q=75 1x, /_next/image?url=%2Fimg.jpg&w=640&q=75 2x"
            style={
              Object {
                "color": "transparent",
              }
            }
            width={258}
          />
          <div
            className="mx-3 text-center"
          >
            <div
              className="info flex justify-between my-4 mx-3"
            >
              <div
                className="text-left"
              >
                <h2
                  className="font-semibold"
                >
                  Moog Synthesizer
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Electronics
                </p>
              </div>
              <div>
                <h2
                  className="font-extrabold text-xl"
                >
                  $
                  468.00
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Istanbul / Turkey
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          className="cart mx-3 mb-10 border rounded-lg shadow-lg"
        >
          <img
            alt="Moog Synthesizer"
            className="rounded-t-lg shadow-lg w-full"
            data-nimg="1"
            decoding="async"
            height={211.41}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/_next/image?url=%2Fimg.jpg&w=640&q=75"
            srcSet="/_next/image?url=%2Fimg.jpg&w=384&q=75 1x, /_next/image?url=%2Fimg.jpg&w=640&q=75 2x"
            style={
              Object {
                "color": "transparent",
              }
            }
            width={258}
          />
          <div
            className="mx-3 text-center"
          >
            <div
              className="info flex justify-between my-4 mx-3"
            >
              <div
                className="text-left"
              >
                <h2
                  className="font-semibold"
                >
                  Moog Synthesizer
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Two-wheeler
                </p>
              </div>
              <div>
                <h2
                  className="font-extrabold text-xl"
                >
                  $
                  468.00
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Istanbul / Turkey
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          className="cart mx-3 mb-10 border rounded-lg shadow-lg"
        >
          <img
            alt="Moog Synthesizer"
            className="rounded-t-lg shadow-lg w-full"
            data-nimg="1"
            decoding="async"
            height={211.41}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/_next/image?url=%2Fimg.jpg&w=640&q=75"
            srcSet="/_next/image?url=%2Fimg.jpg&w=384&q=75 1x, /_next/image?url=%2Fimg.jpg&w=640&q=75 2x"
            style={
              Object {
                "color": "transparent",
              }
            }
            width={258}
          />
          <div
            className="mx-3 text-center"
          >
            <div
              className="info flex justify-between my-4 mx-3"
            >
              <div
                className="text-left"
              >
                <h2
                  className="font-semibold"
                >
                  Moog Synthesizer
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Two-wheeler
                </p>
              </div>
              <div>
                <h2
                  className="font-extrabold text-xl"
                >
                  $
                  468.00
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Istanbul / Turkey
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          className="cart mx-3 mb-10 border rounded-lg shadow-lg"
        >
          <img
            alt="Moog Synthesizer"
            className="rounded-t-lg shadow-lg w-full"
            data-nimg="1"
            decoding="async"
            height={211.41}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/_next/image?url=%2Fimg.jpg&w=640&q=75"
            srcSet="/_next/image?url=%2Fimg.jpg&w=384&q=75 1x, /_next/image?url=%2Fimg.jpg&w=640&q=75 2x"
            style={
              Object {
                "color": "transparent",
              }
            }
            width={258}
          />
          <div
            className="mx-3 text-center"
          >
            <div
              className="info flex justify-between my-4 mx-3"
            >
              <div
                className="text-left"
              >
                <h2
                  className="font-semibold"
                >
                  Moog Synthesizer
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Two-wheeler
                </p>
              </div>
              <div>
                <h2
                  className="font-extrabold text-xl"
                >
                  $
                  468.00
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Istanbul / Turkey
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          className="cart mx-3 mb-10 border rounded-lg shadow-lg"
        >
          <img
            alt="Moog Synthesizer"
            className="rounded-t-lg shadow-lg w-full"
            data-nimg="1"
            decoding="async"
            height={211.41}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/_next/image?url=%2Fimg.jpg&w=640&q=75"
            srcSet="/_next/image?url=%2Fimg.jpg&w=384&q=75 1x, /_next/image?url=%2Fimg.jpg&w=640&q=75 2x"
            style={
              Object {
                "color": "transparent",
              }
            }
            width={258}
          />
          <div
            className="mx-3 text-center"
          >
            <div
              className="info flex justify-between my-4 mx-3"
            >
              <div
                className="text-left"
              >
                <h2
                  className="font-semibold"
                >
                  Moog Synthesizer
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Two-wheeler
                </p>
              </div>
              <div>
                <h2
                  className="font-extrabold text-xl"
                >
                  $
                  468.00
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Istanbul / Turkey
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          className="cart mx-3 mb-10 border rounded-lg shadow-lg"
        >
          <img
            alt="Moog Synthesizer"
            className="rounded-t-lg shadow-lg w-full"
            data-nimg="1"
            decoding="async"
            height={211.41}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/_next/image?url=%2Fimg.jpg&w=640&q=75"
            srcSet="/_next/image?url=%2Fimg.jpg&w=384&q=75 1x, /_next/image?url=%2Fimg.jpg&w=640&q=75 2x"
            style={
              Object {
                "color": "transparent",
              }
            }
            width={258}
          />
          <div
            className="mx-3 text-center"
          >
            <div
              className="info flex justify-between my-4 mx-3"
            >
              <div
                className="text-left"
              >
                <h2
                  className="font-semibold"
                >
                  Moog Synthesizer
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Two-wheeler
                </p>
              </div>
              <div>
                <h2
                  className="font-extrabold text-xl"
                >
                  $
                  468.00
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Istanbul / Turkey
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          className="cart mx-3 mb-10 border rounded-lg shadow-lg"
        >
          <img
            alt="Moog Synthesizer"
            className="rounded-t-lg shadow-lg w-full"
            data-nimg="1"
            decoding="async"
            height={211.41}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/_next/image?url=%2Fimg.jpg&w=640&q=75"
            srcSet="/_next/image?url=%2Fimg.jpg&w=384&q=75 1x, /_next/image?url=%2Fimg.jpg&w=640&q=75 2x"
            style={
              Object {
                "color": "transparent",
              }
            }
            width={258}
          />
          <div
            className="mx-3 text-center"
          >
            <div
              className="info flex justify-between my-4 mx-3"
            >
              <div
                className="text-left"
              >
                <h2
                  className="font-semibold"
                >
                  Moog Synthesizer
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Two-wheeler
                </p>
              </div>
              <div>
                <h2
                  className="font-extrabold text-xl"
                >
                  $
                  468.00
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Istanbul / Turkey
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          className="cart mx-3 mb-10 border rounded-lg shadow-lg"
        >
          <img
            alt="Moog Synthesizer"
            className="rounded-t-lg shadow-lg w-full"
            data-nimg="1"
            decoding="async"
            height={211.41}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/_next/image?url=%2Fimg.jpg&w=640&q=75"
            srcSet="/_next/image?url=%2Fimg.jpg&w=384&q=75 1x, /_next/image?url=%2Fimg.jpg&w=640&q=75 2x"
            style={
              Object {
                "color": "transparent",
              }
            }
            width={258}
          />
          <div
            className="mx-3 text-center"
          >
            <div
              className="info flex justify-between my-4 mx-3"
            >
              <div
                className="text-left"
              >
                <h2
                  className="font-semibold"
                >
                  Moog Synthesizer
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Electronics
                </p>
              </div>
              <div>
                <h2
                  className="font-extrabold text-xl"
                >
                  $
                  468.00
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Istanbul / Turkey
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          className="cart mx-3 mb-10 border rounded-lg shadow-lg"
        >
          <img
            alt="Moog Synthesizer"
            className="rounded-t-lg shadow-lg w-full"
            data-nimg="1"
            decoding="async"
            height={211.41}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/_next/image?url=%2Fimg.jpg&w=640&q=75"
            srcSet="/_next/image?url=%2Fimg.jpg&w=384&q=75 1x, /_next/image?url=%2Fimg.jpg&w=640&q=75 2x"
            style={
              Object {
                "color": "transparent",
              }
            }
            width={258}
          />
          <div
            className="mx-3 text-center"
          >
            <div
              className="info flex justify-between my-4 mx-3"
            >
              <div
                className="text-left"
              >
                <h2
                  className="font-semibold"
                >
                  Moog Synthesizer
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Electronics
                </p>
              </div>
              <div>
                <h2
                  className="font-extrabold text-xl"
                >
                  $
                  468.00
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Istanbul / Turkey
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          className="cart mx-3 mb-10 border rounded-lg shadow-lg"
        >
          <img
            alt="Moog Synthesizer"
            className="rounded-t-lg shadow-lg w-full"
            data-nimg="1"
            decoding="async"
            height={211.41}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/_next/image?url=%2Fimg.jpg&w=640&q=75"
            srcSet="/_next/image?url=%2Fimg.jpg&w=384&q=75 1x, /_next/image?url=%2Fimg.jpg&w=640&q=75 2x"
            style={
              Object {
                "color": "transparent",
              }
            }
            width={258}
          />
          <div
            className="mx-3 text-center"
          >
            <div
              className="info flex justify-between my-4 mx-3"
            >
              <div
                className="text-left"
              >
                <h2
                  className="font-semibold"
                >
                  Moog Synthesizer
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Electronics
                </p>
              </div>
              <div>
                <h2
                  className="font-extrabold text-xl"
                >
                  $
                  468.00
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Istanbul / Turkey
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          className="cart mx-3 mb-10 border rounded-lg shadow-lg"
        >
          <img
            alt="Moog Synthesizer"
            className="rounded-t-lg shadow-lg w-full"
            data-nimg="1"
            decoding="async"
            height={211.41}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/_next/image?url=%2Fimg.jpg&w=640&q=75"
            srcSet="/_next/image?url=%2Fimg.jpg&w=384&q=75 1x, /_next/image?url=%2Fimg.jpg&w=640&q=75 2x"
            style={
              Object {
                "color": "transparent",
              }
            }
            width={258}
          />
          <div
            className="mx-3 text-center"
          >
            <div
              className="info flex justify-between my-4 mx-3"
            >
              <div
                className="text-left"
              >
                <h2
                  className="font-semibold"
                >
                  Moog Synthesizer
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Electronics
                </p>
              </div>
              <div>
                <h2
                  className="font-extrabold text-xl"
                >
                  $
                  468.00
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Istanbul / Turkey
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          className="cart mx-3 mb-10 border rounded-lg shadow-lg"
        >
          <img
            alt="Moog Synthesizer"
            className="rounded-t-lg shadow-lg w-full"
            data-nimg="1"
            decoding="async"
            height={211.41}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/_next/image?url=%2Fimg.jpg&w=640&q=75"
            srcSet="/_next/image?url=%2Fimg.jpg&w=384&q=75 1x, /_next/image?url=%2Fimg.jpg&w=640&q=75 2x"
            style={
              Object {
                "color": "transparent",
              }
            }
            width={258}
          />
          <div
            className="mx-3 text-center"
          >
            <div
              className="info flex justify-between my-4 mx-3"
            >
              <div
                className="text-left"
              >
                <h2
                  className="font-semibold"
                >
                  Moog Synthesizer
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Electronics
                </p>
              </div>
              <div>
                <h2
                  className="font-extrabold text-xl"
                >
                  $
                  468.00
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Istanbul / Turkey
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          className="cart mx-3 mb-10 border rounded-lg shadow-lg"
        >
          <img
            alt="Moog Synthesizer"
            className="rounded-t-lg shadow-lg w-full"
            data-nimg="1"
            decoding="async"
            height={211.41}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/_next/image?url=%2Fimg.jpg&w=640&q=75"
            srcSet="/_next/image?url=%2Fimg.jpg&w=384&q=75 1x, /_next/image?url=%2Fimg.jpg&w=640&q=75 2x"
            style={
              Object {
                "color": "transparent",
              }
            }
            width={258}
          />
          <div
            className="mx-3 text-center"
          >
            <div
              className="info flex justify-between my-4 mx-3"
            >
              <div
                className="text-left"
              >
                <h2
                  className="font-semibold"
                >
                  Moog Synthesizer
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Electronics
                </p>
              </div>
              <div>
                <h2
                  className="font-extrabold text-xl"
                >
                  $
                  468.00
                </h2>
                <p
                  className="font-extralight text-xs"
                >
                  Istanbul / Turkey
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</section>
`;
